README.md

```diff
@@ -32,34 +40,28 @@ Start by cloning this repo in your local setup:
 git clone git@github.com:maticnetwork/avail-light.git
 ```
 
-Create one yaml configuration file in the root of the project & put following content:
+Create one yaml configuration file in the root of the project & put following content. Config example is for a bootstrap client, detailed config specs can be found bellow.
 
 ```bash
 touch config.yaml
 ```
 
 ```yaml
+log_level = "info"
 http_server_host = "127.0.0.1"
-http_server_port = 7000
+http_server_port = "7000"
 
 ipfs_seed = 1
-ipfs_port = 37000
+ipfs_port = "37000"
 ipfs_path = "avail_ipfs_store"
 
-# put full_node_rpc = https://testnet.polygonavail.net/rpc incase you are connecting to devnet
 full_node_rpc = ["http://127.0.0.1:9933"]
-# put full_node_ws = wss://testnet.polygonavail.net/ws incase you are connecting to devnet
 full_node_ws = ["ws://127.0.0.1:9944"]
-# None in case of default Light Client Mode
 app_id = 0
-
 confidence = 92.0
 avail_path = "avail_path"
-
-bootstraps = [["12D3KooWMm1c4pzeLPGkkCJMAgFbsfQ8xmVDusg272icWsaNHWzN", "/ip4/127.0.0.1/tcp/39000"]]
-
-# See https://docs.rs/log/0.4.14/log/enum.LevelFilter.html for possible log level values
-log_level = "INFO"
+prometheus_port = 9520
```

Comment:
nit: are we putting numbers in quotes?

Reply:
@prabal-banerjee depends on the way we parse configs. IPFS port has quotes because of the custom parser, but prometheus_port doesn't need one.